docs/sbpy/data/dataclass.rst

```diff
@@ -402,37 +431,50 @@ object:
 
 or if you want to add a column to your object:
 
+.. doctest-requires:: astropy>=5
+
     >>> from astropy.table import Column
     >>> obs.table.add_column(Column(['V', 'V', 'R', 'i'], name='filter'))
-    >>> obs  # doctest: +SKIP
+    >>> obs
     <QTable length=4>
         ra       dec          t       filter
        deg       deg                        
      float64   float64       Time      str1 
     --------- --------- ------------- ------
-    10.223423 -12.42123  2451523.6234      V
-    10.233453 -12.41562  2451523.7345      V
-    10.243452 -12.40435  2451523.8525      R
+    10.323423 -12.42123  2451523.6234      V
+    10.333453 -12.41562  2451523.7345      V
+    10.343452 -12.40435  2451523.8525      R
      10.25546  -12.3946 2451523.94653      i
 
 The same result can be achieved using the following syntax:
 
-    >>> obs['filter2'] = ['V', 'V', 'R', 'i']  # doctest: +SKIP
-    >>> obs  # doctest: +SKIP
+.. doctest-requires:: astropy>=5
+
+    >>> obs['filter2'] = ['V', 'V', 'R', 'i']
+    >>> obs
     <QTable length=4>
         ra       dec          t       filter filter2
        deg       deg                                
      float64   float64       Time      str1    str1 
     --------- --------- ------------- ------ -------
-    10.223423 -12.42123  2451523.6234      V       V
-    10.233453 -12.41562  2451523.7345      V       V
-    10.243452 -12.40435  2451523.8525      R       R
+    10.323423 -12.42123  2451523.6234      V       V
+    10.333453 -12.41562  2451523.7345      V       V
+    10.343452 -12.40435  2451523.8525      R       R
      10.25546  -12.3946 2451523.94653      i       i
 
 Similarly, existing columns can be modified using:
 
-    >>> obs['filter'] = ['g', 'i', 'R', 'V']  # doctest: +SKIP
-    
+.. testsetup::
+
+    >>> from astropy.table import Column
+    >>> if len(obs) < 4:  # then this is astropy<5
```

Comment:
Here obs is modified for astropy<5. I wonder why not having doctest-requires directive to make this dependency more explicit as in other places but perhaps there is a good reason to do this that I have missed.